javascript/javascript1/week4/Voice-Assistant.js

```diff
@@ -0,0 +1,86 @@
+let vname = '';
+const toDo = [];
+function getReply(command) {
+  const arrayOfStrings = command.split(" ");
+  
+  if (command.startsWith("Hello my name is")) {
+    vname = arrayOfStrings[4];
+    return `Nice to meet you ${vname}.` ;
+  }
+  if (command =="What is my name?") {
+    return `Your name is ${vname}.`;
+  }
+  if (command == "Add fishing to my todo") {
```

Comment:
Here also do the same as you did for the name. Check that all of the todos command start with `Add ` and ends with ` to my todo`, right? So you could make it more dynamic than this and avoid several `if`s into one.

Reply:
done